File: Leetcode/2011/2011.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int x = 0;
        for (const auto &i : operations) {
            if (i[0] == '-' || i[2] == '-') {
                --x;
            } else {
                ++x;
            }
        }
        return x;
    }
};

File: Leetcode/2011/2011.go
func finalValueAfterOperations(operations []string) int {
	x := 0
	for i := 0; i < len(operations); i++ {
		if operations[i][0] == '-' || operations[i][2] == '-' {
			x--
		} else {
			x++
		}
	}
	return x
}